Fix this error using "--disable-libstdcxx-dual-abi" flag:
error: 'const __sso_string {aka const class std::basic_string<char>}' has no member named '_M_s'; did you mean '_M_rep'?
    &((const std::__sso_string*) that)->_M_s._M_p));

diff -Naur a/libstdc++-v3/src/c++11/cow-stdexcept.cc b/libstdc++-v3/src/c++11/cow-stdexcept.cc
--- a/libstdc++-v3/src/c++11/cow-stdexcept.cc	2016-01-18 21:09:55.000000000 +0100
+++ b/libstdc++-v3/src/c++11/cow-stdexcept.cc	2016-03-07 19:48:51.170749365 +0100
@@ -292,6 +292,7 @@
   return (const char*) txnal_read_ptr((void**)&bs->_M_dataplus._M_p);
 }
 
+#if _GLIBCXX_USE_DUAL_ABI
 const char*
 _txnal_sso_string_c_str(const void* that)
 {
@@ -299,6 +300,7 @@
       (void* const*)const_cast<char* const*>(
 	  &((const std::__sso_string*) that)->_M_s._M_p));
 }
+#endif
 
 void
 _txnal_cow_string_D1_commit(void* data)
